### PR TITLE
Simpler esy setup

### PR DESCRIPTION
**Pull request by ulrikstrid**
You don't have to use a resolution for esy to work

### CODE REVIEW

**Comment by rgrinberg** [comment]
cc @prometheansacrifice

**Comment by prometheansacrifice** [comment]
I'm used to overriding unpublished opam packages with resolutions, but this seems fine as well. I guess it's an unnecessary to have resolutions in most cases.

Off-chance, if some one does include ocaml-lsp in the their dependencies erroneously, resolutions will prevent solver errors. Just think, this should be pointed out. In any case, such complaints should be reported here at ocamllsp, but could be reported/discussed in esy channels

**Comment by andreypopp** [comment]
By the way sha commit isn’t necessary if specifying a dependency or dev dependency , it will resolve it to the latest and record resolution into a lock file for next installs.

File: README.md [hunk 1]
```diff
@@ -27,9 +27,6 @@ project's `package.json`:
 
 ```
   "devDependencies": {
-    "@opam/ocaml-lsp-server": "*"
-  },
-  "resolutions": {
     "@opam/ocaml-lsp-server": "github:ocaml/ocaml-lsp:ocaml-lsp-server.opam#<git-commit-sha>"
```
[the remark this answers is not in the document]
Comment (prometheansacrifice) [reply README.md:30]: This suggestion can be checked in and merged

**Pull request merged**